src/image.rs

```diff
@@ -89,6 +83,20 @@ impl<Img> Map<Img> {
         id
     }
 
+    /// Replaces the given image in the map if it exists. Returns the image or None.
+    /// Note: Calling this will trigger a redraw the next time `Ui::draw_if_changed` is called.
```

Comment:
Would you mind adding another line break before the `Note:` lines on this method, the `insert` method and the `remove` method docs? It's just that `rustdoc` normally doesn't add a line break when rendering unless there's at least one blank line in between.